editor/limbo_ai_editor_plugin.cpp

```diff
@@ -1464,12 +1464,17 @@ LimboAIEditor::LimboAIEditor() {
 	EDITOR_SETTINGS()->add_property_hint(PropertyInfo(Variant::INT, "limbo_ai/editor/layout", PROPERTY_HINT_ENUM, "Classic:0,Widescreen Optimized:1"));
```

Comment:
This looks the same as the code below (some of it, anyway). I suggest moving this line outside of conditional compiling, and just leaving inside code that differs.

Reply:
Just checked. godot::EditorSettings.add_property_hint is not available through godot-cpp. At least its not in my generated bindings straight from godot-cpp@master. Is this one of the reasons you did the precompiler differentiation in the first place? How do you want me to proceed?

Reply:
Right, for some reason I read those as the same method :facepalm: